modules/op-release-notes-1-19.adoc

```diff
@@ -23,9 +23,9 @@ In addition to fixes and stability improvements, the following sections highligh
 apiVersion: triggers.tekton.dev/v1beta1
 kind: EventListener
 metadata:
-  name: example
+  name: listener-tolerations
```

Comment:
```suggestion
  name: listener-securitycontext
```
we can use above name which is related to sample

Reply:
Fixed.